src/main/java/org/worshipsongs/importer/SongDao.java

```diff
@@ -11,27 +11,34 @@
 /**
  * Created by Pitchu on 10/19/2015.
  */
-public class SongDao {
-    public int getSongId(Connection connection, String title)
+public class SongDao implements ISongDao
+{
+    private Connection connection;
+    public SongDao(Connection connection)
+    {
+        this.connection = connection;
+    }
+
+    public Song getSongId(Song song)
```

Comment:
Consider renaming the method as simply "getId". Consider changing the method signature as below:
```
public int getId(String title)
{
...
}
```